presto-main/src/main/java/com/facebook/presto/execution/Output.java

```diff
@@ -29,16 +30,19 @@
     private final ConnectorId connectorId;
     private final String schema;
     private final String table;
+    private final ConnectorCommitHandle connectorCommitHandle;
```

Comment:
Again, would suggest using `String` and call it `serializedCommitOutput`

Reply:
OK, will try that soon. Thanks.